src/OT.js

```diff
@@ -43,6 +43,7 @@ const removeNativeEvents = (events) => {
   const eventNames = Object.keys(events);
   OT.removeNativeEvents(eventNames);
   each(events, (eventHandler, eventType) => {
+    nativeEvents.removeAllListeners(eventType);
     nativeEvents.removeListener(eventType, eventHandler);
```

Comment:
from my perspective the call to `removeAllListeners` just should happen once per `eventType` - nevertheless the call on the next line is obsolete, isn't it?

Reply:
I agree, we should remove line 47. @abdulajet any thoughts?